File: 11 - Currency Conversion/Python/main.py
import requests

rate = requests.get("https://api.exchangeratesapi.io/latest").json()["rates"]["GBP"]

nEUR = float(input("How many euros are you exchanging? €"))
print("€{} at an exchange rate of {} is £{}.".format(nEUR, rate, nEUR * rate))

File: 11 - Currency Conversion/Haskell/main.hs
main :: IO()

main = do
    putStrLn "How many euros are you exchaging?"
    nEUR <- readLn :: IO Float
    putStrLn "What is the exchange rate?"
    rate <- readLn :: IO Float

    putStrLn (show nEUR ++ " euros at a rate of " ++ show rate ++ " is " ++ show (nEUR * rate) ++ " pounds.")